internal/runbits/git/test/integration/git_test.go

```diff
@@ -79,7 +79,7 @@ func (suite *GitTestSuite) AfterTest(suiteName, testName string) {
 }
 
 func (suite *GitTestSuite) TestEnsureCorrectProject() {
-	err := gitlet.EnsureCorrectProject("test-owner", "test-project", filepath.Join(suite.dir, constants.ConfigFileName), "test-repo", outputhelper.NewCatcher(), blackhole.New())
+	err := runbitsGit.EnsureCorrectProject("test-owner", "test-project", filepath.Join(suite.dir, constants.ConfigFileName), "test-repo", outputhelper.NewCatcher(), blackhole.New())
```

Comment:
Nit, could probably just use `git` as the package name

Reply:
I tried, but https://github.com/ActiveState/cli/pull/2866/files#diff-de5d57791df0ffc2779a980eed8972e44d069c7d6bbe4f59a2912b3dfcfd6cf6R13 uses `git` :(